SimpleServlet/src/main/java/com/skankhunt220/repository/UserRepository.java

```diff
@@ -0,0 +1,56 @@
+package main.java.com.skankhunt220.repository;
+
+import static com.mongodb.client.model.Filters.eq;
+
+import org.bson.Document;
+import org.bson.types.ObjectId;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientException;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import main.java.com.skankhunt220.entities.User;
+
+public class UserRepository {
+	private MongoClient mc;
+	private MongoDatabase database;
+	private MongoCollection<Document> collection;
+
+	public UserRepository() {
+		mc = new MongoClient("localhost", 27017);
+		database = mc.getDatabase("UserDB");
+		collection = database.getCollection("users");
+	}
+
+	public void Create(User user) {
+		collection.insertOne(new Document("_id", new ObjectId()).append("firstName", user.getFirstName())
+				.append("middleName", user.getMiddleName()).append("lastName", user.getLastName()));
+		closeConnection();
```

Comment:
When you are done with using your **`Connection`**, you need to explicitly close it by calling its `close()` method or use `try()` in order to release any other database resources (**cursors, handles, etc**) the connection may be holding on to. Rewrite it without closing manually. Example: 
```
public void create(User user) throws SQLException {
		try (Connection connection = DriverManager.getConnection(url, userSql, password)) {
			try (PreparedStatement preparedStatement = connection.prepareStatement(INSERT_USER_SQL)) {
				prepareSqlQuery(preparedStatement, user).execute();
			}
		}
	}
```